sql/pipelines/src/test/scala/org/apache/spark/sql/pipelines/graph/MaterializeTablesSuite.scala

```diff
@@ -262,6 +262,7 @@ abstract class MaterializeTablesSuite extends BaseCoreExecutionTest {
   test("invalid schema merge") {
     val session = spark
     import session.implicits._
+    implicit def sqlContext: org.apache.spark.sql.classic.SQLContext = spark.sqlContext
```

Comment:
it's needed by `MemoryStream`, but I think we should make it take implicit `SparkSession` instead. cc @HeartSaVioR

Reply:
Hmm I don't know why MemoryStream takes SQLContext. Is SQLContext purely for compatibility purpose and SparkSession is a drop-in replacement? If then why not deprecate SQLContext?

Reply:
We've already deprecated SQLContext ...